### PR TITLE
BB2-2380 - New Documentation Overview Section

### PR DESCRIPTION
**Pull request by gmaciolek**
Adds new Overview section to static site documentation

**JIRA Ticket:**
[BB2-2380](https://jira.cms.gov/browse/BB2-2380)

**User Story or Bug Summary:**

New high-level overview targeted toward a general audience, with some awareness of API technology, Medicare, and FHIR. Goal of section is to help the reader quickly learn what the Blue Button API is within 2 minutes. 

### What Does This PR Do?

- Publishes Overview section
- Removes "This is the FHIR R4 (v2) documentation for the Blue Button API." information panel from the beginning of the documentation (this has been replaced by a link to V1 documentation under the Versions subheading in the new Overview section.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

* Make sure there's a working overview link in the left navigation
* Review Overview section for any formatting issues
* Check that all links in the Overview section work

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? **No**.
    * Does this PR modify or invalidate any of our security controls?  **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

* CMSgov/some_repo#42

### Submitter Checklist

I have gone through and verified that...:

* [ x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [ x] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [x ] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x ] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [ x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [ x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: _pages/developers.md [hunk 1]
```diff
@@ -27,14 +28,95 @@ ctas:
...
+
+BB2.0 Version 2 (V2) is based on [FHIR release (R4)](http://hl7.org/fhir/R4/){:target="_blank"} and the [CARIN CDPDE Implementation Guide (Carin for Blue Button)](http://hl7.org/fhir/us/carin-bb/STU1.1/){:target="_blank"} and was released in July 2021\. To learn about migrating to BB2.0 V2, go to [Migrating to V2 FAQ](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Migrating-to-V2-FAQ)
+
+#### Version 1 
+
+Version 1 (V1) is the original Blue Button API, based on [FHIR release 3 (STU3)](http://hl7.org/fhir/STU3/){:target="_blank"}. Development on V1 is limited to bug fixes and basic maintenance. If you're developing a new app, use V2\. For information about BB2.0 V1, see the [V1 Documentation](https://bluebutton.cms.gov/v1/). 
+
+### Environments
+
+We currently maintain production and sandbox environments for developing with the BB2.0 API. For technical information about our environments, see [Base FHIR URLs](https://docs.google.com/document/d/1Ktw_lgOsqZRopJCTV8krJ2W5zjxNoeolxFcHQJje0zw/edit#heading=h.ikw2u81euyqi).
```
Comment (dtisza1) [inline _pages/developers.md:104]: @gmaciolek This link is going to a google doc.

File: _pages/developers.md [hunk 2]
```diff
@@ -27,14 +28,95 @@ ctas:
...
+* Allow patients to share health information like current medications, lab results, and medical imaging with providers..
+* Give caregivers an accurate picture of a Medicare enrollee's health data to facilitate care coordination and communication with providers.
+
+For other examples of how applications use the Blue Button 2.0 API, visit the [Blue Button app directory on Medicare.gov](https://www.medicare.gov/manage-your-health/medicares-blue-button-blue-button-20/blue-button-apps).
+
+### Technical Specifications and Standards 
+
+The Blue Button 2.0 API is a RESTFul API, based on the HL7 FHIR standard and the CARIN Consumer Directed Payer Data Exchange Implementation Guide. It supplies data in JSON format, and uses the OAuth 2.0 protocol for authorization.
+
+* [HL7 FHIR Standard](http://www.hl7.org/fhir/index.html){:target="_blank"}
```
Comment (dtisza1) [inline _pages/developers.md:85]: Depending on the new tabs choice, we may want to remove the `{:target="_blank"}` here and in other locations?

**Review by dtisza1 — CHANGES_REQUESTED** [review]
@gmaciolek This is looking good to me.  Just had a few change requests.

File: _pages/developers.md (hunk 1 above)
Comment (gmaciolek) [reply _pages/developers.md]: Thanks for cathing that. Should be fixed now

**Pull request merged**